README.md

```diff
@@ -1 +1,187 @@
+onfido.document.list("applicant_id")                         # => Returns all applicant's documents
+```
+
+**Note:** The first parameter in the upload method should be an `InputStream` created from the desired file to upload.
+ 
+**Note:** The download method returns the file in the form of a `InputStream`.
+
+#### Live Photos
+
+Live Photos, like documents, can provide supporting evidence for Onfido checks.
+
+```java
+onfido.live_photo.upload(inputStream, "fileName", requestBody) # => Creates a live photo
+onfido.live_photo.find("live_photo_id")                        # => Finds a live photo
+onfido.live_photo.download("live_photo_id")                    # => Downloads a live photo as binary data
+onfido.live_photo.list("applicant_id")                         # => Returns all applicant's live photos
+```
+
+**Note:** The first parameter in the upload method should be an `InputStream` created from the desired file to upload.
+ 
+**Note:** The download method returns the file in the form of a `InputStream`.
+
+#### Checks
+
+Checks are requests for Onfido to check an applicant, by commissioning one or
+more "reports" on them.
+
+```java
+onfido.check.create("applicant_id", reportNamesArray) 
+onfido.check.find("check_id")
+onfido.check.resume("check_id")
+onfido.check.list("applicant_id")
+```
+
+#### Reports
+
+Reports provide details of the results of some part of a "check". They are
+created when a check is created, so the Onfido API only provides support for
+finding and listing them. For paused reports specifically, additional support for resuming and
+ cancelling reports is also available.
+
+```java
+onfido.report.find("report_id")
+onfido.report.list("check_id")
+onfido.report.resume("report_id")
+onfido.report.cancel("report_id")
+```
+
+#### Address Lookups
+
+Onfido provides an address lookup service, to help ensure well-formatted
+addresses are provided when creating "applicants". To search for addresses
+by postcode, use:
+
+```java
+onfido.address.pick("SE1 4NG")
+```
+
+#### Webhook Endpoints
+
+Onfido allows you to set up and view your webhook endpoints via the API, as well
+as through the dashboard.
+
+```java
+onfido.webhook.create(requestBody)        # => Creates a webhook endpoint
+onfido.webhook.find("webhook_id")         # => Finds a single webhook endpoint
+onfido.webhook.list()                     # => Returns all webhook endpoints
+```
+
+#### SDK Tokens
+
+Onfido allows you to generate JSON Web Tokens via the API in order to authenticate
+with Onfido"s [JavaScript SDK](https://github.com/onfido/onfido-sdk-ui).
+
+```java
+onfido.sdk_token.create(applicant_id: "applicant_id", referrer: "referrer")
+```
+
+## Webhooks
+
+Each webhook endpoint has a secret token, generated automatically and [exposed](https://onfido.com/documentation#register-webhook) in the API. When sending a request, Onfido includes a signature computed using the request body and this token in the `X-Signature` header.
```

Comment:
Minor: I think we could refocus this paragraph on the endpoint to generate a webhook specifically.